mars/oscar/backends/ray/backend.py

```diff
@@ -48,8 +48,9 @@ async def create_actor_pool(
         pg = get_placement_group(pg_name) if pg_name else None
         if not pg:
             bundle_index = -1
+        num_cpus = kwargs.get('main_pool_cpus', 0)
         actor_handle = ray.remote(RayMainPool).options(
-            num_cpus=0,  # main pool doesn't do horse work, mark it use no cpu.
+            num_cpus=num_cpus,
```

Comment:
Why does a main pool use more than one cpu?

Reply:
The main pool may use 1 cpu or 0 cpu, no more than 1 cpu.

Reply:
The main pool doesn't do task computing, we can take it using 0 cpu. Or just enforce all ray actors using 1 cpu. What's the situation for main pool to use different cpu values?

Reply:
The rule is,
- Main pool always use 0 CPU.
- The sub pool of supervisor always use 0 CPU.
- The sub pool of worker always use 1 CPU.